.gitignore

```diff
@@ -3,8 +3,9 @@
 ##
 ## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
 
-# Personal custom ignore
+# Pyrohax custom ignores
 out/
+vs/
```

Comment:
There is another .vs/ so can you check why this one is necessary? Is it a generated folder as well?

Reply:
Well, the .vs/ only works on the folder called .vs, the vs folder is what is generated with CMake Visual Studio.

Reply:
Although I realize I could generate the solution in a different folder to avoid annoyances with files trying to be submitted... Will take a look around.